localization/english/character/dvg_character_traits_l_english.yml

```diff
@@ -1,4 +1,3 @@
 ﻿l_english:
- #very_persistent:0 "Persistent"
- #very_persistent_desc:0 "This character is known for their persistence."
- 
\ No newline at end of file
+ immortal:0 "Important"
+ immortal_desc:0 "This character is known for its importance in historical events."
```

Comment:
I think this description isn't perfect, what about 
"This character will not die before the time is right" 
or something, I think it should be clear that it is a "i don't die" trait until a certain event. I think your description works however, so if you want to keep it it is fine.

Reply:
I used this way of describing, as this is how it's done for all traits.
I have not come across the perfect desc yet, but we can always change if we do.